Apsitvarkom.UnitTests/Api/Controllers/CleaningEventControllerTests.cs

```diff
@@ -214,6 +217,111 @@ public async Task GetById_RepositoryThrows_Status500InternalServerErrorReturned(
     }
     #endregion
 
+    #region Update tests
+    public async Task Update_NullIdEntered_ValidationResultsInBadRequestResponseReturned()
```

Comment:
`[Test]` attribute is missing - we don't ever test this method with the current setup.